src/Chatkit.php

```diff
@@ -649,7 +649,8 @@ public function sendMultipartMessage($options)
 
             if (isset($part['file'])) {
                 $attachment_id = $this->uploadAttachment($token, $room_id, $part);
-                $part['file'] = $attachment_id;
+                $part['attachment'] = [ 'id' => $attachment_id ];
+                unset($part['file']);
```

Comment:
It would be better (but not required) to create a fresh object and copy only the relevant keys

Reply:
True. I might change the verify function to return them. I'll do that in a separate PR.